templates/yymmdd-request-for-proposal.md

```diff
@@ -0,0 +1,57 @@
+## Collaborator
```

Comment:
Put the rfp name at the top. Imo this goes at the bottom, but is a nuance and could just be after the title / overview.